exercises/practice/pythagorean-triplet/src/main/scala/PythagoreanTriplet.scala

```diff
@@ -0,0 +1,5 @@
+object PythagoreanTriplet {
+  def isPythagorean(nums: (Int, Int, Int)): Boolean = ???
+  def pythagoreanTriplets(low: Int, high:Int): Seq[(Int, Int, Int)] = ???
+  def pythagoreanTripletsSum(p: Int): Seq[(Int, Int, Int)] = ???
```

Comment:
I see. Clarification of expectations since it seemed confusing. But this feels like a breaking change. I will revert this and fix the tests instead. It seems all other languages I check do not test the ordering of the 3 arguments, so it is up to the implementer to decide if this is a requirement to pass or not. Both interpretations are certainly possible.